### DISCUSSION TITLE
[Grant] Metaplex Earn - Season 1

### DISCUSSION THREAD

**Discussion by MetaplexFoundation** [metaplex-foundation/dao#12] [Community Grant Submissions]
**SUBMISSION DETAILS**

Submitter: Forgd
Email: millan@forgd.com

$MPLX requested: 1,000,000 (with any leftover returned to the DAO)

Title: Metaplex Earn — Season 1

**TL;DR**

We propose to allocate 1,000,000 MPLX tokens to a Metaplex Earn campaign in an effort to deepen the on-chain liquidity for the MPLX token and create reward-generating opportunities for the Metaplex community.

By deepening the on-chain liquidity of the MPLX token, the Metaplex community will benefit from more efficient on-chain price discovery and more seamless trading conditions.

In doing so, this campaign aims to empower the on-chain Solana economy to gain access to the Metaplex DAO and help govern the future of digital assets using MPLX.

**BACKGROUND & RATIONALE**

Metaplex has solidified its position as a cornerstone of the Solana ecosystem, powering virtually all token, memecoin, and NFT creation on the network. Recent record-breaking metrics in November 2024 — $3.5 million in protocol fees, 1.4 million fungible tokens minted, and 879k unique signers — underscore the extraordinary growth of digital assets and the Metaplex protocol as a whole.

Given this surge in growth, it’s imperative to provide opportunities for new and existing community members to gain on-chain exposure to Metaplex’s success via the $MPLX token.

With current on-chain liquidity [depth] standing at ~2.7 million USD (at the time of writing), MPLX’s on-chain liquidity relative to its market capitalization is relatively low against other comparable Solana DeFi protocols, as seen below.

![image](https://github.com/user-attachments/assets/8dd71ac3-07cc-4f57-b35d-1e3991c44809)

Accordingly, we believe that a Metaplex Earn [pilot] campaign, which we can call “Metaplex Earn — Season 1,” focused on deepening on-chain liquidity, is prudent given current market circumstances.

Additionally, the MPLX token has historically lacked DeFi utility and reward-generating avenues, something that this campaign aims to address.

**WHAT DO WE PROPOSE?**

We request a budget of 1,000,000 MPLX tokens from the DAO to be used across a period of 2 months, with any unused MPLX tokens being returned after the conclusion of the campaign.

In these 2 months, up to 1,000,000 MPLX tokens will be allocated as rewards to liquidity providers of MPLX-based pairs across the major Solana DEX(s) and Kamino Finance.

These rewards would be dynamically allocated on a weekly basis based on market conditions and data-driven insights.

Apart from increasing on-chain liquidity, this campaign will also introduce reward-generating avenues for the MPLX community, something, as alluded to above, had been lacking since the inception of the Metaplex DAO.

This campaign can also prove to bring a plethora of auxiliary benefits, including, but not limited to:

**1. Additional DeFi Integrations**: as on-chain MPLX scales up, DeFi protocols like lending markets, derivatives trading, and protocols would be able to integrate MPLX as it poses less risk of accruing bad debt (cross-contagion risk due to shallow on-chain depth) to the respective protocol.

**2. MPLX-Paired LPs**: currently, there are hardly any non-SOL or USDC paired MPLX pools. As the liquidity in these main pools scales up, pairing tokens against the MPLX token becomes feasible for small caps like memecoins, further driving liquidity for the MPLX token and initiating a virtuous flywheel.

**3. On-chain DAO Treasury Participation**: DAOs (or other entities) that operate solely on-chain will be able to accumulate MPLX as a strategic reserve asset for their treasuries to participate in the governance of the Metaplex ecosystem in a more seamless manner without needing to tap into CEX liquidity. Moreover, DAO treasury swaps with the Metaplex DAO can also become more attractive and feasible.

This pilot campaign effectively aims to test the waters of the Solana-Metaplex economy, so the requested MPLX budget is relatively low. As we progress through the pilot campaign and gain insights as users reveal their preferences, future campaigns can be more methodology-oriented based on past data.

Furthermore, as on-chain liquidity increases and volumes follow suit, the marginal need for external incentives in MPLX decreases as trading fees can sustain the existing base of liquidity providers. However, this still warrants a period of external rewards (e.g., this campaign and a future one with a broadened scope) in the interim to deepen liquidity.

**Comment by jindouyunz** [comment]
Why not pay to list on major CEX (Binance, Upbit, e.g.)?
It cost 1 million dollars on average. The Metaplex Earn costs about 0.28 million dollars every 2 months.
CEX listing is considerably a better choice.

**Comment by MetaplexFoundation** [comment]
Voting is live in the Metaplex DAO until January 2, 2025. 
https://dao.metaplex.com/dao/Metaplex/proposal/8c2278d6T1VQ4hHZwx4B79b4B5NyKFoZhRnB91CRjSAH

**Comment by MetaplexFoundation** [comment]
This grant proposal was passed by the Metaplex DAO.
